Dockerfile

```diff
@@ -1,56 +1,90 @@
-FROM php:8.0-cli as rector
-WORKDIR /rector
+ARG PHP_VERSION=8.0
+FROM php:${PHP_VERSION}-cli as base
+
+RUN apt-get update && apt-get install -y \
+    libzip4 \
+    libicu63 \
+    && rm -rf /var/lib/apt/lists/*
+
+FROM base as build
+
+WORKDIR /build
 
 # Install php extensions
 RUN apt-get update && apt-get install -y \
-        git \
-        unzip \
         g++ \
-        libzip-dev \
+        git \
```

Comment:
Lets keep it like this, but in future it is always great to keep refactoring for separate MR than adding new feature so it is easier to review.